cmd/restic/cmd_sync.go

```diff
@@ -99,7 +112,8 @@ func (ft *FileTracker) CompleteItem(item string, previous, current *restic.Node,
 func runSync(opts SyncOptions, gopts GlobalOptions, term *termstatus.Terminal, args []string) error {
 	ctx := gopts.ctx
 
-	fileTracker := FileTracker{}
+	start := time.Now()
```

Comment:
We can probably drop these time.Now()  trackers that we were using to record the time various steps took.